tests/integration/docusaurus/expectations/auto_initializing_expectations/auto_initializing_expect_column_mean_to_be_between.py

```diff
@@ -92,7 +93,11 @@
 )
 assert expectation_result["expectation_config"]["kwargs"]["column"] == "trip_distance"
 assert expectation_result["expectation_config"]["kwargs"]["auto"] is True
-assert expectation_result["expectation_config"]["kwargs"]["min_value"] == 2.83
-assert expectation_result["expectation_config"]["kwargs"]["max_value"] == 3.06
+assert 2.83 == approx(
+    expectation_result["expectation_config"]["kwargs"]["min_value"], 0.01
+)
```

Comment:
Using an approximate assertion instead.

Reply:
What are the actual values? The default estimator is `exact`, so this should be deterministic.